app/src/main/java/com/hedvig/app/feature/offer/ui/OfferAdapter.kt

```diff
@@ -210,19 +188,11 @@ class OfferAdapter(
                                             }
                                         )
                                     }
-
-                                    // Remove campaign views if campaign type is unknown
-                                    if (
-                                        incentive.asFreeMonths == null &&
-                                        incentive.asMonthlyCostDeduction == null &&
-                                        incentive.asNoDiscount == null &&
-                                        incentive.asPercentageDiscountMonths == null
-                                    ) {
-                                        premiumContainer.background = null
-                                        campaign.remove()
-                                    }
                                 } ?: run {
                                 discountButton.setText(R.string.OFFER_ADD_DISCOUNT_BUTTON)
+                                val buttonTextColor =
+                                    ContextCompat.getColor(discountButton.context, R.color.textColorLink)
```

Comment:
```suggestion
                                    discountButton.context.colorAttr(R.attr.textColorLink)
```

Requires the same import as before.

Generally, when working with themes, avoid using direct references to colors and prefer instead to use theme attributes - this allows us to support light/dark in a more maintainable fashion.

This will also be true when moving to compose, where we will refer to colors from `MaterialTheme` 😄

Reply:
Good point. I always mix these up 🙈